packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -140,13 +146,11 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
       </Drawer.Header>
 
       <Drawer.Body>
-        <PanelGroup accordion>
-          <Panel header={t('authors.panels.name')} defaultExpanded>
```

Comment:
I haven't tested it locally to see the difference but don't we need defaultExpanded or the header??

Reply:
i removed the accordion because it was of no use. Now it shows everything at once.